src/game/gamemodes/StealTheCrown.js

```diff
@@ -0,0 +1,250 @@
+        crown = player;
+        crown.graphic.addChild(this.crownimg);
+        this.restTime = 0;
+        const score = this.game.scoreManager.getScore('Steals', idTag);
+        this.game.scoreManager.setScore('Steals', idTag, score + 1);
+      }
+    });
+
+    // Place them in the middle of the arena for now
+    circle.x = this.arenaCenterX;
+    circle.y = this.arenaCenterY;
+  }
+
+  // Clean up after the gamemode is finished.
+  cleanUp() {
+    this.arenaGraphic.destroy();
+  }
+
+  // Called when an entity is respawned.
+  onPlayerRespawn(entity) {
+    // Move the entity close to the center
+    entity.x = this.arenaCenterX + Math.cos(Math.random() * Math.PI * 2) * this.respawnArea;
+    entity.y = this.arenaCenterY + Math.sin(Math.random() * Math.PI * 2) * this.respawnArea;
+  }
+
+  onWindowResize() {
+    // Updates the playing field to fit the window. Might be a better way to do this.
+
+    const width = this.game.gameStageWidth * 0.5;
+    const height = this.game.gameStageHeight * 0.5;
+    this.topLine.y = -height;
+    this.bottomLine.y = height;
+    this.rightLine.x = width;
+    this.leftLine.x = -width;
+
+    // make sure crownentity always stays within borders on resize
+    if (this.crownEntity.x < -width) {
+      this.crownEntity.x = -width + CROWNWIDTH;
+    } else if (this.crownEntity.x > width) {
+      this.crownEntity.x = width - CROWNWIDTH;
+    }
+    if (this.crownEntity.y < -height) {
+      this.crownEntity.y = -height + CROWNHEIGHT;
+    } else if (this.crownEntity.y > height) {
+      this.crownEntity.y = height - CROWNHEIGHT;
+    }
+    // Puts a player back into the playing field if the window resizes
+    Object.keys(this.players).forEach(id => {
+      const entity = this.players[id];
+      if (!entity.playerLeft) {
+        if (entity.x < -width) {
+          entity.x = -width + entity.radius;
+        } else if (entity.x > width) {
+          entity.x = width - entity.radius;
+        }
+        if (entity.y < -height) {
+          entity.y = -height + entity.radius;
+        } else if (entity.y > height) {
+          entity.y = height - entity.radius;
+        }
+      } else if (!entity.dead) {
+        if (entity === crown) {
+          crown.graphic.removeChild(this.crownimg);
+          crown.die();
+          this.crownEntity.x = 0;
+          this.crownEntity.y = 0;
+          this.addCrown();
+        } else entity.die();
+      }
+    });
+  }
+
+  static getResources() {
+    return [{ name: 'crown', path: 'passthe/crown.png' }];
+  }
+
+  static getConfig() {
+    return {
+      joinPhase: 2,
+      playerRadius: 32,
+      backgroundColor: 0x061639,
+      leave: {
+        // If a player has the bomb then the bomb must pop before they are removed
+        removeTime: 5,
+      },
+      respawn: {
+        time: 1,
+        phase: 2,
+      },
+      rules: [
+        'Steal the crown! Hold it to get points, and steal it from whoever currently carries it!',
```

Comment:
Name of the game mode is not needed. This is already displayed on the UI and controller.